services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -42,10 +44,7 @@
 import com.linkedin.venice.listener.ServerReadMetadataRepository;
 import com.linkedin.venice.listener.ServerStoreAclHandler;
 import com.linkedin.venice.listener.StoreValueSchemasCacheService;
-import com.linkedin.venice.meta.ReadOnlyLiveClusterConfigRepository;
-import com.linkedin.venice.meta.ReadOnlySchemaRepository;
-import com.linkedin.venice.meta.ReadOnlyStoreRepository;
-import com.linkedin.venice.meta.StaticClusterInfoProvider;
+import com.linkedin.venice.meta.*;
```

Comment:
Please don't import everything. IntelliJ does this automatically, which is a bit annoying.